.chloggen/fix_omissions_in_receiver_mysql_remodeling_metrics.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/mysql
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove the order by clause for the column that does not exist
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [32693,33271]
```

```suggestion
issues: [33271]
```
I think we can just keep the issue that's being fixed, it has enough context and information for what went wrong and what's being fixed 👍

---

I see. I will do that.